dali/operators/reader/video_reader_decoder_cpu_op.cc

```diff
@@ -68,6 +81,10 @@ even in the variable frame rate scenario.)code")
   .AddArg("sequence_length",
       R"code(Frames to load per sequence.)code",
       DALI_INT32)
+  .AddOptionalArg("enable_frame_num",
```

Comment:
```suggestion
  .AddOptionalArg("enable_frame_idx",
```
?

Reply:
I want to keep it consistent with [the video reader ](https://docs.nvidia.com/deeplearning/dali/user-guide/docs/operations/nvidia.dali.fn.readers.video.html).

Reply:
I understand and agree - nonetheless, it's unfortunate.